### PR TITLE
Update Spanish translation

### PR DESCRIPTION
**Pull request by regiregi22**
This is an update for the stringtables that lacked Spanish translation.

## What type of PR is this.
3. [X ] Enhancement

### What have you changed and why?
Information: I have added the stringtables for the Spanish language
    

### Please verify the following and ensure all checks are completed.

1. [ ] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ X] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: Load the mod and check that the strings are translated to spanish

********************************************************
Notes:

### CODE REVIEW

**Comment by Bob-Murphy** [comment]
I changed it to merge into the `unstable` branch as this is our work-branch.

**Review by Bob-Murphy — CHANGES_REQUESTED** [review]
Please note that you used a different spaces on the stringtables of core, gui and maps than they initially have, hence basically overwriting every line rather than only the ones you worked with.
Please adapt, thank you.

**Comment by regiregi22** [comment]
> Please note that you used a different spaces on the stringtables of core, gui and maps than they initially have, hence basically overwriting every line rather than only the ones you worked with. Please adapt, thank you.

I just used tabler as the translation tool, so it used the standard spacing format. I will try to change that.

**Review by Bob-Murphy — APPROVED** [review]

File: A3A/addons/core/Stringtable.xml [hunk 1]
```diff
@@ -5018,31 +5025,35 @@
...
         <Czech>&lt;br/&gt; - opraveno</Czech>
         <Korean>&lt;br/&gt; - 수리되었습니다</Korean>
+        <Spanish>&lt;br/&gt; - reparado</Spanish>
       </Key>
       <Key ID="STR_A3A_base_vehicleBoxRestore_refueled">
         <Original>&lt;br/&gt; - refueled</Original>
         <Russian>&lt;br/&gt; - заправлена</Russian>
         <Czech>&lt;br/&gt; - natankovat</Czech>
         <Korean>&lt;br/&gt; - 재급유되었습니다</Korean>
+        <Spanish>&lt;br/&gt; - repostado</Spanish>
```
Comment (ncermesoni) [inline A3A/addons/core/Stringtable.xml:5056]:
```suggestion
        <Spanish>&lt;br/&gt; - han sido repostados</Spanish>
```

File: A3A/addons/core/Stringtable.xml [hunk 2]
```diff
@@ -5078,30 +5091,35 @@
         <Russian>Загрузка сервера завершена.</Russian>
         <Polish>Zakończono ładowanie serwera.</Polish>
         <Portuguese>Carregamento servidor concluida</Portuguese>
+        <Spanish>Completada carga del servidor.</Spanish>
```
Comment (ncermesoni) [inline A3A/addons/core/Stringtable.xml:5094]:
```suggestion
        <Spanish>Completada la carga del servidor.</Spanish>
```

File: A3A/addons/core/Stringtable.xml [hunk 3]
```diff
@@ -5130,160 +5150,191 @@
...
         <Original>This garrison is full, choose another zone to add this HC squad into it's garrison.</Original>
         <Russian>Этот гарнизон полон, выберите другую локацию для того чтобы добавить этот отряд в гарнизон.</Russian>
         <Korean>이 주둔지는 꽉 찼으므로, 이 하이 커맨드 분대를 주둔지에 추가하려면 다른 구역을 선택하십시오.</Korean>
+        <Spanish>La guarnición está llena, selecciona otra zona para añadir esta escuadra de HC a su guarnición.</Spanish>
       </Key>
       <Key ID="STR_A3A_garrison_exceed_limit">
         <Original>Adding this squad to garrison will exceed garrison limit. Some of them will join garrison, the rest will be dismissed and their cost will be refunded.</Original>
         <Russian>Добавление этого отряда в гарнизон превысит лимит гарнизона. Какая-то их часть присоединится к гарнизону, оставшиеся будут распущены, а их стоимость будет возмещена.</Russian>
         <Korean>이 분대를 주둔지에 추가하면 주둔군 제한을 초과하게 됩니다. 그들 중 일부는 주둔지에 합류할 것이고, 나머지는 해산되며 비용은 환불될 것입니다.</Korean>
+        <Spanish>Añadir esta escuadra a la guarnición excederá el límite de la guarnición. Algunos de ellos se unirán a la guartnición, el resto serán descartados y su coste será reembolsado.</Spanish>
```
Comment (ncermesoni) [inline A3A/addons/core/Stringtable.xml:5198]:
```suggestion
        <Spanish>Añadir esta escuadra a la guarnición excederá el límite de la guarnición. Algunos de ellos se unirán a la guarnición, el resto serán descartados y su coste será reembolsado.</Spanish>
```

File: A3A/addons/core/Stringtable.xml [hunk 4]
```diff
@@ -5130,160 +5150,191 @@
...
       </Key>
       <Key ID="STR_A3A_garrison_reached_limit">
         <Original>Garrison limit has reached, you can't add new units anymore.</Original>
         <Russian>Достигнут лимит гарнизона, вы больше не можете добавлять в него новые юниты.</Russian>
         <Korean>주둔지가 한계에 도달했으며, 더 이상 새 유닛을 추가할 수 없습니다.</Korean>
+        <Spanish>La guarnición ha alcanzado su límite, no puedes añadir más unidades nuevas.</Spanish>
```
Comment (ncermesoni) [inline A3A/addons/core/Stringtable.xml:5204]:
```suggestion
        <Spanish>La guarnición ha alcanzado su límite, no puedes añadir nuevas unidades.</Spanish>
```

File: A3A/addons/core/Stringtable.xml [hunk 5]
```diff
@@ -5130,160 +5150,191 @@
...
       </Key>
       <Key ID="STR_A3A_garrison_select_zone">
         <Original>Select the zone on where you will send the selected troops as garrison.</Original>
         <Russian>Выберите зону, куда вы отправите выбранные войска в качестве гарнизона.</Russian>
         <Korean>선택한 병력을 주둔지로 보낼 구역을 선택하십시오.</Korean>
+        <Spanish>Selecciona la zona hacia la que vas a enviar los tropas seleccionadas como guarnición.</Spanish>
```
Comment (ncermesoni) [inline A3A/addons/core/Stringtable.xml:5210]:
```suggestion
        <Spanish>Selecciona la zona en la que vas a guarnicionar los tropas seleccionadas.</Spanish>
```
https://dle.rae.es/guarnicionar?m=form

File: A3A/addons/core/Stringtable.xml [hunk 6]
```diff
@@ -5130,160 +5150,191 @@
...
       </Key>
       <Key ID="STR_A3A_GCTracker_tracker_notification">
         <Original>Last Garbage Clean was %1 ago. Keep track of it.</Original>
         <Korean>마지막 쓰레기 청소는 %1 전에 실행되었습니다. 계속 기록합니다.</Korean>
+        <Spanish>La ultima Limpieza de Basura fue hace %1. Mantenlo supervisado.</Spanish>
       </Key>
       <Key ID="STR_A3A_GCTracker_tracker_ran_gc">
         <Original>"Garbage Cleaner Tracker run a garbage clean because the time passed the threshold. Keep track of GC times."</Original>
         <Korean>"쓰레기 청소 기록 프로그램이 임계값을 초과하여 쓰레기를 청소합니다. 쓰레기 청소 시간을 계속 기록합니다."</Korean>
+        <Spanish>"El Seguimiento del Limpiador de Basura ejecuta un limpiador a causa de superar el umbral de tiempo. Manten supervisado el Limpiador de Basura."</Spanish>
```
Comment (ncermesoni) [inline A3A/addons/core/Stringtable.xml:5323]:
```suggestion
        <Spanish>"El Seguimiento del Limpiador de Basura llevó a cabo una limpieza porque el umbral de tiempo fue superado. Mantén supervisado el Limpiador de Basura."</Spanish>
```

File: A3A/addons/gui/Stringtable.xml [hunk 7]
```diff
@@ -591,47 +591,59 @@
...
       </Key>
       <Key ID="STR_antistasi_dialogs_buy_vehicle_refuel_tooltip">
         <Original>This item can refuel others\n%1 max fuel is %2</Original>
         <Korean>이 아이템은 다른 차량을 재급유할 수 있습니다.\n%1의 최대 연료량은 %2입니다.</Korean>
+        <Spanish>Este objeto puede repostar a otros \nEl máximo combustible de %1 es %2</Spanish>
       </Key>
       <Key ID="STR_antistasi_dialogs_buy_vehicle_loot_tooltip">
         <Original>This item can collect loot\n%1 max load is %2</Original>
         <Korean>이 아이템은 전리품을 수집할 수 있습니다.\n%1의 처ㅣ대 수용량은 %2입니다.</Korean>
+        <Spanish>Este objeto puede recolectar el loot \nLa carga máxima de %1 es %2</Spanish>
```
Comment (ncermesoni) [inline A3A/addons/gui/Stringtable.xml:634]:
```suggestion
        <Spanish>Este objeto puede recolectar loot \nLa carga máxima de %1 es %2</Spanish>
```

File: A3A/addons/gui/Stringtable.xml [hunk 8]
```diff
@@ -2926,6 +2942,7 @@
         <Polish>Zapisane Rozgrywki</Polish>
         <Portuguese>Jogos salvos</Portuguese>
         <Chinesesimp>保存游戏</Chinesesimp>
+        <Spanish>Partidas Guardados</Spanish>
```
Comment (ncermesoni) [inline A3A/addons/gui/Stringtable.xml:2945]:
```suggestion
        <Spanish>Partidas Guardadas</Spanish>
```

File: A3A/addons/gui/Stringtable.xml [hunk 9]
```diff
@@ -2959,6 +2978,7 @@
         <Polish>Skopiuj Starą Rozgrywkę</Polish>
         <Portuguese>copiar jogo antigo</Portuguese>
         <Chinesesimp>加载之前的存档</Chinesesimp>
+        <Spanish>Copiar Partida antigua</Spanish>
```
Comment (ncermesoni) [inline A3A/addons/gui/Stringtable.xml:2981]:
```suggestion
        <Spanish>Copiar Partida existente</Spanish>
```

File: A3A/addons/gui/Stringtable.xml [hunk 10]
```diff
@@ -2970,6 +2990,7 @@
         <Polish>Wczytaj Stare Parametry</Polish>
         <Portuguese>Copiar parametros anteriores</Portuguese>
         <Chinesesimp>复制之前的参数</Chinesesimp>
+        <Spanish>Cargar parámetros antiguos</Spanish>
```
Comment (ncermesoni) [inline A3A/addons/gui/Stringtable.xml:2993]:
```suggestion
        <Spanish>Cargar parámetros existentes</Spanish>
```

**Review by ncermesoni — COMMENTED** [review]
Leaving some suggestions, as a native spanish speaker. Great job overall!

**Comment by regiregi22** [comment]
Great suggestions [ncermesoni](https://github.com/ncermesoni), thank you for that! Un abrazo.

**Pull request merged**